src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -230,7 +230,7 @@ export const ArtworkApp: React.FC<Props> = props => {
 
       <Spacer y={6} />
 
-      <RelatedWorksQueryRenderer slug={artwork.slug} />
+      {/* <RelatedWorksQueryRenderer slug={artwork.slug} /> */}
```

Comment:
Maybe add a `// TODO` here giving some context?

Reply:
Unclear what the TODO is! Need to confirm this has the desired effect, then [revisit](https://artsy.slack.com/archives/CA8SANW3W/p1677177993873459?thread_ts=1676999056.487519&cid=CA8SANW3W) for a solution. We should know more very soon.

Reply:
np! Just didn't want this comment to be lost later should other things get picked up. Something like "This line seems to impact performance in ES" would be sufficient.